### PR TITLE
Add OTLP JSON payload examples

### LINKED ISSUE #462: Add examples to OTLP/JSON
OTLP/JSON is a human readable format. It would be nice to have some example payloads in the OTLP specification to demonstrate what it can look like. Examples sometimes are very useful when you want to understand the structure of the payload.

See for example a user asking for it: https://github.com/open-telemetry/opentelemetry-specification/issues/2773#issuecomment-1245573473

### PR DESCRIPTION
**Pull request by joaopgrassi**
Fixes #462

I was not sure where to add the files, so I just went with some initial idea. I'm open to suggestions on where to put them/refer to them. 

I tested it with a local collector and all works. 

cc @tigrannajaryan

### CODE REVIEW

**Review by arminru — APPROVED** [review]

File: specification/otlp-json-examples/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# OTLP JSON request examples
+
+This folder contains a collection of OTLP JSON files for all signals
```
Comment (tigrannajaryan) [inline specification/otlp-json-examples/README.md:3]:
```suggestion
This folder contains a collection of example OTLP JSON files for all signals
```

**Review by bogdandrutu — COMMENTED** [review]
Because I know that we will do manual work here withe these examples. should we have at least a check that they are valid JSONs?

**Review by bogdandrutu — COMMENTED** [review]
Because I know that we will do manual work here withe these examples. should we have at least a check that they are valid JSONs?

**Review by bogdandrutu — COMMENTED** [review]
Because I know that we will do manual work here withe these examples. should we have at least a check that they are valid JSONs?

**Comment by tigrannajaryan** [comment]
Yes more checks would be useful. Related: we have a PR coming that adds link checking: https://github.com/open-telemetry/opentelemetry-proto/pull/459

**Comment by tigrannajaryan** [comment]
> I was also thinking if we shouldn't publish a json schema for OTLP. That way the guess work of writing OTLP JSON is removed and we could also check it against the schema in a gh action. I saw some tool to generate JSON schema from proto files but didn't have time to check. If you think it's valid, I can create issues and maybe work on it.

I would not want to maintain another schema definition. The Protobuf definitions are the source of truth. If we could generate JSON schema from Protobufs that would be great.

**Review by jsuereth — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
Merging. @joaopgrassi it would be great if you could follow up with a PR to add JSON checks.

**Comment by joaopgrassi** [comment]
Will do!

**Comment by chalin** [comment]
Hi all, I've posted a suggestion to:

- #471

Comments are welcome.

**Pull request merged**